src/main/java/ch/bfh/bti7081/s2018/green/models/entities/Doctor.java

```diff
@@ -0,0 +1,5 @@
+package ch.bfh.bti7081.s2018.green.models.entities;
+
+public class Doctor {
```

Comment:
This class doesn't exist. I will add the Staff class in another pull request so you can delete this class from this pull request.

Reply:
I will just change that in my pull request later.. So you can leave it that way.